src/docs/product/user-feedback/index.mdx

```diff
@@ -1,25 +1,76 @@
 ---
 title: User Feedback
-sidebar_order: 120
+sidebar_order: 65
 description: "Learn how you can view user feedback submissions which, paired with the original event, give you additional insight into issues."
 ---
 
-While event data allows you to collect information about errors in your application, getting any extra helpful feedback can be crucial in determining the root cause of an issue. Sentry provides the ability to collect additional information with User Feedback.
+Sentry automatically detects errors thrown by your application, such as performance issues and user experience problems like rage clicks. But there are other frustrations your users may encounter (broken permission flows, broken links, typos, misleading UX, business logic flaws, and so on).
 
-The **User Feedback** page in [sentry.io](https://sentry.io) displays feedback submissions from users who experienced an error while using your application. Each submission includes the name and email address entered by the user, as well as their description of what happened to produce the error:
+That’s where Sentry’s User Feedback Widget and Crash-Report Modal features come in. Setting up one or both, lets you collect feedback from your end-users when they run into bugs in your application. Sentry then links that feedback to available rich debugging context, including errors, replays, and tags such as URL, allowing you to get to the root of the problem faster.
 
-![User Feedback page with submissions.](user-feedback.png)
+For example, using Session Replay with the User Feedback Widget could save you time, helping you quickly understand the context of the user problem by seeing exactly what they experienced before submitting their feedback.
 
-In each submission, you can click the “View Event” link, which takes you to the **Issue Details** page for that error event (or instance of the issue). From there, you can dig into what caused the issue and start working to resolve it:
+## User Feedback Widget
 
-![The event that generated the user feedback on the Issue Details page.](user-feedback-event.png)
+The User Feedback Widget is an out-of-the-box, persistent widget you can add anywhere on your web application. It allows users to submit feedback quickly and easily any time they encounter something that isn’t working as expected. Learn more about how to install and customize it [here](https://github.com/getsentry/sentry-javascript/tree/develop/packages/feedback#setup).
+
+![User Feedback widget.](user-feedback-widget.png)
```

Comment:
This image looks good but I think we can re-size it to be way smaller.
Also, is there any way to avoid the purple border? It doesn't need it @lizokm @vivianyentran